e2e2/test/images/nvidia/gpu_unit_tests/tests/test_sysinfo.sh

```diff
@@ -63,7 +63,7 @@ test_nvidia_gpu_throttled()
 
     # https://docs.nvidia.com/deploy/nvml-api/group__nvmlClocksEventReasons.html#group__nvmlClocksEventReasons
     # The only  bit allowed is nvmlClocksEventReasonGpuIdle 0x0000000000000001LL
-    filter="egrep -v -e '(0x0000000000000000|0x0000000000000001)'"
+    filter="egrep -v -e '(0x0000000000000000|0x0000000000000001|0x0000000000000004)'"
```

Comment:
dumb question, what does this mean?

Reply:
It is an issue found by Niki. Where there are some GPUs don't get overcooked, and it will throttle.